spec/normal_user.api_key

```diff
@@ -0,0 +1 @@
+kP/U4BBQV0s2TcwN2MWwFrbwGDyMGkZEFt0era1yJc9eSuBhC+PiaDF1L+eylxsEoOzrnIsd4LgYOGQcIP6Y
```

Comment:
Not sure it's needed to store this in a file.

Reply:
Should it just be generated on the fly as a constant then?

Reply:
It will be better than reading to a file